cm_page_builder-rails.gemspec

```
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
   spec.test_files = Dir["spec/**/*"]
 
-  spec.add_dependency "rails", "~> 6.0.0"
+  spec.add_dependency "rails", "~> 6.1.5"
```

@DeboDevelop - Instead of hard coding the specific rails version we should check if the project is using any rails 6.0+, it would give more flexibility to the gem.

@manikandan0603 @mikevic @anbublacky - If we are moving with this we need to check if it works with all the available versions for rails >= 6.0 even rails 7.

I feel by hard coding we are forcing the project to update to the desired version which the page builder uses even though it is not the latest stable version. This would make the life of a developer difficult and for us in maintaining the gem as well.
```suggestion
  spec.add_dependency 'rails', '>=6.0'
```

---

@AdityaTiwari2102 - this suggest is a good add. I don't think locking to `6.1.15` is a great idea. However, not really sure about `Rails 7` 😅